internal/uvm/create.go

```diff
@@ -316,9 +368,36 @@ func Create(opts *UVMOptions) (*UtilityVM, error) {
 	}
 
 	uvm.hcsSystem = hcsSystem
+	defer func() {
+		if err != nil {
+			uvm.Close()
+		}
+	}()
+
+	if uvm.operatingSystem == "linux" {
+		uvm.gcslog, err = uvm.listenVsock(linuxLogVsockPort)
+		if err != nil {
+			return nil, err
+		}
+	}
+
 	return uvm, nil
 }
 
+func (uvm *UtilityVM) listenVsock(port uint32) (net.Listener, error) {
```

Comment:
Really? I didn't think this worked without explicitly writing the service entry to the registry. Did the credential story change here?

Reply:
For HCS-created VMs, you just have to set the security descriptor appropriately when creating the VM. The registry is not consulted.

Reply:
Good to know.

Reply:
I added a comment at the place where we set the security descriptor.